src/psyclone/tests/psyir/backend/c_test.py

```diff
@@ -364,17 +366,49 @@ def test_cw_binaryoperator():
         assert cwriter(binary_operation) == expected
 
     # Test that an unsupported operator raises a error
-    # pylint: disable=abstract-method, too-few-public-methods
-    class Unsupported():
+    # pylint: disable=too-few-public-methods
+    class Unsupported(object):
         '''Dummy class'''
-    # pylint: enable=abstract-method, too-few-public-methods
+        def __init__(self):
+            pass
+    # pylint: enable=too-few-public-methods
     binary_operation._operator = Unsupported
     with pytest.raises(VisitorError) as err:
         _ = cwriter(binary_operation)
     assert "The C backend does not support the '" in str(err)
     assert "' operator." in str(err)
 
 
+def test_cw_loop():
+    '''Tests writing out a Loop node in C. It parses Fortran code
+    and outputs it as C.'''
```

Comment:
Please add a note that this is currently a literal translation and that, as a consequence, the resulting C code is not functionally equivalent to the Fortran. Please reference the new Issue you created for this.

Reply:
Done.